pluto/src/vis/schematic/Forms.tsx

```diff
@@ -672,6 +659,12 @@ export const ButtonTelemForm = ({ path }: { path: string }): ReactElement => {
       <Input.Item label="Output Channel" grow>
         <Channel.SelectSingle value={sink.channel} onChange={handleSinkChange} />
       </Input.Item>
+      <Form.NumericField
+        label="Delay"
```

Comment:
What do you think about calling it "Activation Delay" or "Hold Threshold" instead? I don't think "Delay" is descriptive enough, it wouldn't be immediately obvious to somebody that is playing around with a schematic and plops down a button. We could also add a tooltip when hovering over the label that gives a description that says "How long the button must be held before the command is sent" or something similar.

Reply:
Renamed to activation delay. We can add those tooltip helpers in a separate issue